functions/api/wb/settings/server/rules.ts

```diff
@@ -0,0 +1,70 @@
+import type { Env } from 'wildebeest/backend/src/types/env'
+import type { ContextData } from 'wildebeest/backend/src/types/context'
+import * as errors from 'wildebeest/backend/src/errors'
+import { type Database, getDatabase } from 'wildebeest/backend/src/database'
+
+export const onRequestGet: PagesFunction<Env, any, ContextData> = async ({ env, request }) => {
+	return handleRequestPost(getDatabase(env), request)
+}
+
+export async function handleRequestGet(db: Database) {
+	const query = `SELECT * from server_rules;`
+	const result = await db.prepare(query).all<{ id: string; text: string }>()
+
+	if (!result.success) {
+		return new Response('SQL error: ' + result.error, { status: 500 })
+	}
+
+	return new Response(JSON.stringify(result.results ?? []), { status: 200 })
+}
+
+export const onRequestPost: PagesFunction<Env, any, ContextData> = async ({ env, request }) => {
+	return handleRequestPost(getDatabase(env), request)
+}
+
+export async function handleRequestPost(db: Database, request: Request) {
+	// TODO: validate the request! only the instance admin should be able to edit this
+	const authenticated = true
```

Comment:
@xtuc @jculvey not totally sure what should I do here 😓

Reply:
does this need the admin implementation first? 🤔

Reply:
After a quick chat with @jculvey I guess that we need to:
 - implement the admin flag (or whatever we want to use)
 - get the current JWT from the request
 - get the admin via SQL
 - check that the current JWT's email corresponds to the admin's one
 
 how does that shound @xtuc?

Reply:
I would write the login in this way:
...
 - get the current JWT from the request
-  verify the JWT
 - load the actor by email from the JWT email
 - check that actor is admin